feature/githubRepoSearch/routes/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+
+const router = express.Router();
+
+/* GET home page. */
```

Comment:
This comment can be avoided.

Reply:
I used express-generator and this was the default boilerplate. I have removed the comments now.